validation/POIs_validation.py

```diff
@@ -0,0 +1,71 @@
+import geopandas as gpd
+import numpy as np
+import matplotlib.pyplot as plt
+from geopy.distance import great_circle
+
+official_filename = "Restaurant_and_Food_Related.shp"
+OSM_filename = "belfast_gb_2019_1600m_buffer.gpkg"
+POIs_name = "fresh_food_market"
+
+gdf = gpd.read_file(OSM_filename, layer='destinations')
+OSM_points = gdf[gdf['dest_name']==POIs_name]
+
+OPoints = gpd.GeoDataFrame.from_file(official_filename)
+print("number of OSM points: " ,len(OSM_points))
+print("number of Official points: ", len(OPoints))
+
+#Calculate number of OSM_points intersect with OPoints

```

Comment:
Make sure you check that the two GeoDataFrame crs attributes are equal to each other before proceeding with spatial analysis. If they differ, project one to the other's CRS.

Reply:
Moved the projection up before intersections